src/style.css

```diff
@@ -1 +1,125 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 900;
+  font-style: normal;
+  font-display: swap;
+}
+
+/* FONTS */
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't set margins/paddings, etc to all elements, reset only for the tags that you actually need. For example, div/span doesn't have any margins or padding, so you don't need to reset them